client/dashboard/sites/add-new-site/index.tsx

```diff
@@ -0,0 +1,134 @@
+import { recordTracksEvent } from '@automattic/calypso-analytics';
+import { WordPressLogo, JetpackLogo } from '@automattic/components';
+import { formatNumber } from '@automattic/number-formatters';
+import {
+	__experimentalHStack as HStack,
+	__experimentalVStack as VStack,
+	__experimentalText as Text,
+	Button,
+} from '@wordpress/components';
+import { __, sprintf } from '@wordpress/i18n';
+import { download, reusableBlock, Icon } from '@wordpress/icons';
+import devSiteBanner from 'calypso/assets/images/a8c-for-agencies/dev-site-banner.svg';
+import { preventWidows } from 'calypso/lib/formatting';
+import Column from './column';
+import MenuItem from './menu-item';
+import './style.scss';
+
+const TRACK_SOURCE_NAME = 'sites-dashboard';
+
+const wordpressClick = () => {
+	recordTracksEvent( 'calypso_sites_dashboard_new_site_action_click_add' );
+	recordTracksEvent( 'calypso_sites_dashboard_new_site_action_click_item', {
+		action: 'wordpress',
+	} );
+};
+const jetpackClick = () => {
+	recordTracksEvent( 'calypso_sites_dashboard_new_site_action_click_jetpack' );
+	recordTracksEvent( 'calypso_sites_dashboard_new_site_action_click_item', {
+		action: 'jetpack',
+	} );
+};
+const migrateClick = () => {
+	recordTracksEvent( 'calypso_sites_dashboard_new_site_action_click_item', {
+		action: 'migrate',
+	} );
+};
+const importClick = () => {
+	recordTracksEvent( 'calypso_sites_dashboard_new_site_action_click_import' );
+	recordTracksEvent( 'calypso_sites_dashboard_new_site_action_click_item', {
+		action: 'import',
+	} );
+};
+const offerClick = () => {
+	recordTracksEvent( 'calypso_sites_dashboard_new_site_action_click_item', {
+		action: 'offer',
+	} );
+};
+
+function AddNewSite() {
+	return (
+		<HStack alignment="flex-start" style={ { padding: '16px' } } spacing={ 4 }>
+			<Column title={ __( 'Add new site' ) }>
+				<MenuItem
+					icon={ <WordPressLogo /> }
+					title={ __( 'WordPress.com' ) }
```

Comment:
We don't localize brand names do we? Seems like an unnecessary string to localize.